extensions-core/mysql-metadata-storage/src/test/java/org/apache/druid/metadata/storage/mysql/MySQLConnectorTest.java

```diff
@@ -57,16 +55,14 @@ public void testIsExceptionTransientMySql()
         new MySQLConnectorSslConfig(),
         MYSQL_DRIVER_CONFIG
     );
-    Assert.assertTrue(connector.connectorIsTransientException(new MySQLTransientException()));
-    Assert.assertTrue(connector.connectorIsTransientException(new MySQLTransactionRollbackException()));
     Assert.assertTrue(
         connector.connectorIsTransientException(new SQLException("some transient failure", "s0", 1317))
     );
     Assert.assertFalse(
         connector.connectorIsTransientException(new SQLException("totally realistic test data", "s0", 1337))
     );
     // this method does not specially handle normal transient exceptions either, since it is not vendor specific
-    Assert.assertFalse(
```

Comment:
why is it changed from assertFalse to assertTrue

Reply:
Thanks for your time first.
+ We changed  MYSQL_TRANSIENT_EXCEPTION_CLASS_NAME from  "com.mysql.jdbc.exceptions.MySQLTransientException" to  "java.sql.SQLTransientException"

+ And in mysql-connector-j-8.2.0.jar, we do not have MySQLTransientException.